src/TLuaInterpreter.cpp

```diff
@@ -346,8 +346,8 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
 {
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     if( ! pHost ) {
-        lua_pushstring( L, tr( "raiseEvent: NULL Host pointer - something is wrong!" )
-                        .toUtf8().constData() );
+        lua_pushstring(L, QStringLiteral("raiseEvent: NULL Host pointer - something is wrong!")
+                       .toLatin1().constData());
```

Comment:
I think that makes sense, because we don't want a QString for Lua, and we're not dealing with UTF-8 here.